### PR TITLE
Test add new os(Racom)

### PR DESCRIPTION
**Pull request by erotel**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Hi im test add new os (Racom):
first problem: I do not see uptime on the device
second problem: device does not support outoctets and intoctets (manufacturer promised repair)

[discovery](https://pastebin.com/Wsd0J4Vb)
[poller](https://pastebin.com/HSPSXnGn)

---
This change is [Reviewable](https://reviewable.io/reviews/librenms/librenms/7400)

### CODE REVIEW

**Comment by erotel** [comment]
One more problem: the device has a slow CPU. when using snmp v2 (snmpbulkwalk), the device can not answer queries. The manufacturer recommended using v1 or snmpbulkwalk with -t5. is one of the options set?

File: includes/polling/os/racomos.inc.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+$hardware = trim(snmp_get($device, '.1.3.6.1.2.1.1.5', '-OQv', '', ''), '"');
+$version  = trim(snmp_get($device, '.1.3.6.1.4.1.33555.1.1.2.2', '-OQv', '', ''), '"');
```
Comment (laf) [inline includes/polling/os/racomos.inc.php]: Can you change these two snmp_gets into multi gets, see `includes/polling/os/netguardian.inc.php` for an example.

File: tests/snmpsim/racomos.snmprec [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Microwave Link
```
Comment (laf) [inline tests/snmpsim/racomos.snmprec]: Is this the genuine sysDescr from a device?

File: includes/definitions/racomos.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+os: racomos
```
[the remark this answers is not in the document]
Comment (erotel) [reply includes/definitions/racomos.yaml]: there are ray and ray2, the firmware is different for each version, but for all drives of the same version (ray2-10GHz, ray2-24GHz, etc.)
A listing of CLI ray2 is "uname -a
Linux RAy2-10L 3.15.7 # 36 Tue Aug 25 09:35:09 Local time zone must be set - see GNU / Linux "
For RAY (v1) CLI does not support uname.

File: tests/snmpsim/racomos.snmprec (hunk 2 above)
Comment (erotel) [reply tests/snmpsim/racomos.snmprec]: not either RAY - Microwave Connection or Microwave Link I'm not done yet, I'm testing it

File: includes/definitions/racomos.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (laf) [reply includes/definitions/racomos.yaml]: Ok so then this should be two OS', RAy and RAy2

File: tests/snmpsim/racomos.snmprec (hunk 2 above)
Comment (laf) [reply tests/snmpsim/racomos.snmprec]: Ok so this PR isn't ready?

File: includes/definitions/racomos.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (erotel) [reply includes/definitions/racomos.yaml]: how to detect which system is ray and ray2?
snmp ray:
root@speedtest:~# snmpwalk -v1 -c public 10.133.7.252
iso.3.6.1.2.1.1.1 = STRING: "RAy - Microwave link"
iso.3.6.1.2.1.1.2 = OID: iso.3.6.1.4.1.33555.1.1
iso.3.6.1.2.1.1.3 = Timeticks: (152433277) 17 days, 15:25:32.77
iso.3.6.1.2.1.1.5 = STRING: "RAY10_H"
iso.3.6.1.2.1.1.6 = STRING: " "
iso.3.6.1.2.1.1.7 = INTEGER: 8
iso.3.6.1.2.1.2.1 = INTEGER: 3

snmp ray2:
root@speedtest:~# snmpwalk -v1 -c public 10.133.69.19
iso.3.6.1.2.1.1.1 = STRING: "RAy - Microwave Link"
iso.3.6.1.2.1.1.2 = OID: iso.3.6.1.4.1.33555.1.1
iso.3.6.1.2.1.1.3 = Timeticks: (152432700) 17 days, 15:25:27.00
iso.3.6.1.2.1.1.5 = STRING: "RAy2-10L"
iso.3.6.1.2.1.1.6 = STRING: " "
iso.3.6.1.2.1.1.7 = INTEGER: 8
iso.3.6.1.2.1.2.1 = INTEGER: 3

File: tests/snmpsim/racomos.snmprec (hunk 2 above)
Comment (erotel) [reply tests/snmpsim/racomos.snmprec]: no, I needed to solve the problems mentioned above

File: includes/definitions/racomos.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (laf) [reply includes/definitions/racomos.yaml]: From that info you really can't.

Maybe it just needs to be one for now with the name RAy

File: tests/snmpsim/racomos.snmprec (hunk 2 above)
Comment (laf) [reply tests/snmpsim/racomos.snmprec]: Ok

**Comment by erotel** [comment]
im test your config [https://gist.githubusercontent.com/laf/5669d72c44d175434f3d6aede8ba3e57/raw/9ff12a39d64f6dce1e7493a6f0636fa6ea691153/gistfile1.txt](url)

but not work.

[discovery.php -h *** -d -m sensors](https://pastebin.com/XMSHJ9aB)

**Comment by laf** [comment]
I'd need to see the actual snmpwalk data but I think this might be because the walk isn't returning an index to use on the end. If that's the case then you'll need to switch back to the php files you created.

**Comment by erotel** [comment]
[snmp-ray](https://pastebin.com/uGAmSbT8)

Pls delete this PR.
Im create new PR correctly.

**Comment by laf** [comment]
We can't delete it, you can only close it and delete your branch. However you can just push more commits to fix it.

**Comment by erotel** [comment]
Ok.
Im go back php files in sensors.
I just can not find out why it does not show uptime.

**Pull request closed without merging**